src/languages/elixir.js

```diff
@@ -278,6 +271,7 @@ export default function(hljs) {
 
   return {
     name: 'Elixir',
+    aliases: ['ex', 'exs'],
```

Comment:
Are `ex` still textual code vs binary?

Reply:
Both `ex` and `exs` are source-code files (text)